### PR TITLE
Publish Docker images

### LINKED ISSUE #114: updated release process
Including Docker image publishing from CI

### PR DESCRIPTION
**Pull request by anyaelena**
- Automate publishing of Docker images to nrel/alfalfa-worker and nrel/alfalfa-web.  
- All commits to develop branch are pushed to Dockerhub with `develop` tag.
- All commits to main are published with a version tag defined [here](https://github.com/NREL/alfalfa/blob/develop/pyproject.toml). (@kbenne @nllong agree that this is where we keep the version?)  `latest` is also updated
- Updates URIs in .env for web and worker images
- Adds VERSION_TAG to .env and replaces hardcoded `latest` in docker-compose.yml

Closes #114

### CODE REVIEW

File: .github/workflows/parse_version.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import toml
```
Comment (nllong) [inline .github/workflows/parse_version.py:1]: nice!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -136,3 +136,33 @@ jobs:
...
+        uses: actions/checkout@v2
+
+      # Install only toml, which is used to extract version.
+      # We don't do anything else with Python in this job.
+      - name: Install toml
+        run: python -m pip install toml
+
+      - name: Build & Publish Images
+        shell: bash
+        run: ./.github/workflows/publish_to_docker.sh
```
Comment (nllong) [inline .github/workflows/ci.yml:165]: this works for now, but we use the GH action script for this in SEED: https://github.com/SEED-platform/seed/blob/develop/.github/workflows/publish.yml#L63-L70. Just an idea for the future.

**Review by nllong — APPROVED** [review]
look marvelous!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (anyaelena) [reply .github/workflows/ci.yml:165]: Thanks!  We build and push two containers based on a bunch of config in the docker-compose, so i didn't see a straightforward action for this.

**Comment by anyaelena** [comment]
@nllong we decided that the pyproject.toml is specific to worker and will end up moved into that dir in #134 and have version removed in #135.  Instead, we'll add a simple version.txt (or json) in the root.

**Pull request merged**